### PR TITLE
updates xml-formatter properties

### PR DESCRIPTION
**Pull request by jam01**
Just updating the default properties after having upgraded the xml-formatter

### CODE REVIEW

**Comment by hazendaz** [comment]
@jam01 Can you rebase this?  I have fixed appveyor and rebase should then come clean.  Thanks.

**Comment by jam01** [comment]
@hazendaz  done

File: src/main/java/net/revelc/code/formatter/xml/XMLFormatter.java [hunk 1]
```diff
@@ -39,13 +39,17 @@ public void init(Map<String, String> options, ConfigurationSource cfg) {
...
         String splitMultiAttrs = options.get("splitMultiAttrs");
+        String wellFormedValidation = options.get("wellFormedValidation");
 
         prefs.setMaxLineLength(maxLineLength != null ? Integer.valueOf(maxLineLength) : null);
         prefs.setWrapLongLines(wrapLongLines != null ? Boolean.valueOf(wrapLongLines) : null);
         prefs.setTabInsteadOfSpaces(tabInsteadOfSpaces != null ? Boolean.valueOf(tabInsteadOfSpaces) : null);
         prefs.setTabWidth(tabWidth != null ? Integer.valueOf(tabWidth) : null);
         prefs.setSplitMultiAttrs(splitMultiAttrs != null ? Boolean.valueOf(splitMultiAttrs) : null);
 
+        if (wellFormedValidation != null)
```
Comment (hazendaz) [inline src/main/java/net/revelc/code/formatter/xml/XMLFormatter.java]: @jam01 Can you add curry brackets around the if statement here?  Then we are good to merge :)

Comment (jam01) [reply src/main/java/net/revelc/code/formatter/xml/XMLFormatter.java]: done :)

Comment (hazendaz) [reply src/main/java/net/revelc/code/formatter/xml/XMLFormatter.java]: Thanks, file needs a format but I'll take this and format afterwards.  Appreciate it.

**Pull request merged**